### DISCUSSION TITLE
RM-TestExternalFiles - Fast and Useful

### DISCUSSION THREAD

**Discussion by ve3meo** [RichardOtter/Genealogy-scripts#5] [General]
Richard, your utility is really effective. I especially appreciate the feature to list those files in a media folder that are not used by the RM database having gotten that in the past only by a complicated combination of Windows commands, the SQLite command line utility and Excel (see https://sqlitetoolsforrootsmagic.com/scrapbook-files-status/). I knew that it would take a higher level language to do it as a seemingly simple utility but lacked the skills and. maybe at the time, the motivation and energy to do so. And your utility also answers other important user questions about their family tree media that are the subject of separate SQLite scripts that I or others have published over the years.

Your utility, whether launched as a Python script or as the compiled .exe, is blazingly fast. The text report output is convenient for working through.

Good job!

**Comment by Stumpy842** [comment]
It appears TestExternalFiles.py does not recognize wildcards in the [IGNORED_OBJECTS] section of the RM-Python-config.ini file. So if I have:

> [IGNORED_OBJECTS]
> FILENAMES =
>   *.txt

I still see any .txt files from my Media folder listed in the report. Is there a way to make this work, please?
